spec/cancancan/model_adapters/mongoid_adapter_spec.rb

```diff
@@ -7,15 +7,13 @@
     end
 
     after(:each) do
-      default_client =
-        if Mongoid::VERSION >= '5.0'
-          Mongoid.default_client
-        else
-          Mongoid.default_session
-        end
-      default_client.collections.reject do |collection|
-        collection.name =~ /system/
-      end.each(&:drop)
+      if Mongoid::VERSION >= '5.0'
+        Mongoid.default_client.collections.reject do |collection|
+          collection.name =~ /system/
+        end.each(&:drop)
+      else
+        Mongoid.default_session.drop
```

Comment:
This looks a bit aggressive, but I needed it to make Mongoid 3.x/4.x builds pass.